### PR TITLE
docs/osf-trivia-list/dasharo.md: add what is Dasharo

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

File: docs/osf-trivia-list/dasharo.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Dasharo is registered trademark and product developed by [3mdeb](https://3mdeb.com).
+
+Dasharo is an open-source firmware distribution focusing on
+[seamless deployment](#dasharo-seamless-deployment),
+[clean and simple code](#dasharo-clean-and-simple-code),
+[long-term maintenance](#dasharo-long-term-maintenance),
+[professional support](#dasharo-support-package),
+[transparent validation](#dasharo-transparent-validation),
+[superior documentation](../index.md)
```
Comment (pietrushnic) [inline docs/osf-trivia-list/dasharo.md:11]: We need section that proves that fact. I saw some community members claiming that our documentation is very good, but I would go to fact that we are strict about providing extending explanation using very known pattern described [here](https://github.com/Dasharo/docs#navigation-menu)

File: docs/index.md [hunk 2]
```diff
@@ -2,15 +2,14 @@
...
-Dasharo is open-source firmware distribution focusing on clean and simple code,
-long-term maintenance, transparent validation, privacy-respecting
-implementation, liberty for the owners, and trustworthiness for all.
-
-Dasharo consists of productized services, open-core, and SaaS products which
-help to provide scalable, modular, easy to combine open-source BIOS, UEFI, and
-firmware solutions. It offers the components that are needed to develop and
-maintain a high quality, and modular firmware, for the stability and security
-of your platform.
+Dasharo is registered trademark and product developed by [3mdeb](https://3mdeb.com).
```
[the remark this answers is not in the document]
Comment (miczyg1) [reply docs/index.md]: https://github.com/Dasharo/docs/pull/499/files#diff-b4d68dc855d0f9476d3f2ee343853bd21bf82ea9960d0cf06661baa244439dd6R14
Done

File: docs/osf-trivia-list/dasharo.md (hunk 1 above)
Comment (miczyg1) [reply docs/osf-trivia-list/dasharo.md:11]: > We need section that proves that fact. I saw some community members claiming that our documentation is very good

Ooof, now I wish I had those messages saved somewhere.

Comment (miczyg1) [reply docs/osf-trivia-list/dasharo.md]: Wasn't sure which adjectives to use so went with `extensive and structured documentation` and the link you provided

File: docs/osf-trivia-list/dasharo.md [hunk 3]
```diff
@@ -1,58 +1,125 @@
...
+(GPG fingerint: `0D5F 6F1D A800 329E B7C5  97A2 ABE1 D0BC 6627 8008`) which
+signs
+	- [Customer Open Source Firmware Release x.y Signing
+Key](https://github.com/3mdeb/3mdeb-secpack/tree/master/customer-keys) (e.g.
+Novacustom Open Source Firmware Release 1.0 Signing Key)
+	- or [dedicated 3mdeb
+keys](https://github.com/3mdeb/3mdeb-secpack/tree/master/dasharo) to given
+platform.
+
+	Keys can be found in [3mdeb-secpack](https://github.com/3mdeb/3mdeb-secpack) repository.
```
Comment (miczyg1) [inline docs/osf-trivia-list/dasharo.md:91]:
```suggestion
    - [CEO/Founder](https://github.com/3mdeb/3mdeb-secpack/blob/master/keys/owner-key/piotr-krol-key.asc)
(GPG fingerint: `E030 9B2D 85A6 7E84 6329  E34B B2EE 71E9 67AA 9E4C`) which
signs
    - [3mdeb Master
Key](https://github.com/3mdeb/3mdeb-secpack/blob/master/keys/master-key/3mdeb-master-key.asc)
(GPG fingerint: `1B57 85C2 965D 84CF 85D1  652B 4AFD 81D9 7BD3 7C54`) which
signs
    - [3mdeb Dasharo Master
Key](https://github.com/3mdeb/3mdeb-secpack/blob/master/dasharo/3mdeb-dasharo-master-key.asc)
(GPG fingerint: `0D5F 6F1D A800 329E B7C5  97A2 ABE1 D0BC 6627 8008`) which
signs
    - [Customer Open Source Firmware Release x.y Signing
Key](https://github.com/3mdeb/3mdeb-secpack/tree/master/customer-keys) (e.g.
Novacustom Open Source Firmware Release 1.0 Signing Key)
    - or [dedicated 3mdeb
keys](https://github.com/3mdeb/3mdeb-secpack/tree/master/dasharo) to given
platform.

    Keys can be found in [3mdeb-secpack](https://github.com/3mdeb/3mdeb-secpack) repository.
```

mkdocs uses 4 spaces for indents IIRC. Hard TABs should be avoided

Comment (pietrushnic) [reply docs/osf-trivia-list/dasharo.md:91]: grrr, my new dev vm setup messed up with editor configuration

**Review by pietrushnic — APPROVED** [review]
@miczyg1 I guess we can merg it?

**Pull request merged**